README.dev.md

```diff
@@ -6,11 +6,612 @@ You will need a machine with at least 2GB of RAM available (see
 https://github.com/zulip/zulip/issues/32 for a plan for how to
 dramatically reduce this requirement).
```

Comment:
This isn't really related to your changes, but I think we can s/2GB/1GB/ at this point and remove the note about reducing the requirement (we addressed that memory usage issue in development, but not production, a few months ago).